app/peptide/payloadstore/payloadstore_test.go

```diff
@@ -17,27 +18,38 @@ func dummyPayload(height int64) *monomer.Payload {
 		GasLimit: &gaslimit,
 	}
 	hash := common.HexToHash(fmt.Sprintf("0x%x", height))
-	return eetypes.NewPayload(&attrs, hash, height)
+
+	return &monomer.Payload{
+		Timestamp:             uint64(attrs.Timestamp),
+		PrevRandao:            attrs.PrevRandao,
+		SuggestedFeeRecipient: attrs.SuggestedFeeRecipient,
+		Withdrawals:           attrs.Withdrawals,
+		NoTxPool:              attrs.NoTxPool,
+		GasLimit:              uint64(*attrs.GasLimit),
+		ParentBeaconBlockRoot: attrs.ParentBeaconBlockRoot,
+		ParentHash:            hash,
+		Height:                height,
+		Transactions:          attrs.Transactions,
+		CosmosTxs:             nil,
+	}
 }
 
 func TestRollback(t *testing.T) {
 	ps := NewPayloadStore()
-	ids := make([]*eetypes.PayloadID, 10)
+	ids := make([]*engine.PayloadID, 10)
 
 	for h := int64(0); h < 10; h++ {
 		payload := dummyPayload(h)
-		require.NoError(t, ps.Add(payload))
-		id, err := payload.GetPayloadID()
-		require.NoError(t, err)
+		ps.Add(payload)
+		id := payload.ID()
```

Comment:
We can remove the extra variable by putting `ids[h] = payload.ID()` on the next line.